dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -689,67 +864,71 @@ data:
           "type": "stat"
         }
       ],
-      "refresh": "1h",
-      "schemaVersion": 37,
-      "style": "dark",
-      "tags": [],
-      "templating": {
-        "list": [
-          {
-            "current": {
-              "selected": true,
-              "text": "rhtap-observatorium-production",
-              "value": "rhtap-observatorium-production"
-            },
-            "hide": 0,
-            "includeAll": false,
-            "multi": false,
-            "name": "datasource",
-            "options": [],
-            "query": "prometheus",
-            "queryValue": "",
-            "refresh": 1,
-            "regex": "/^rhtap-/",
-            "skipUrlSync": false,
-            "type": "datasource"
-          }
-        ]
-      },
-      "time": {
-        "from": "now-28d",
-        "to": "now"
-      },
-      "timepicker": {
-        "refresh_intervals": [
-          "5s",
-          "10s",
-          "30s",
-          "1m",
-          "5m",
-          "15m",
-          "30m",
-          "1h",
-          "2h",
-          "1d"
-        ],
-        "time_options": [
-          "5m",
-          "15m",
-          "1h",
-          "6h",
-          "12h",
-          "24h",
-          "2d",
-          "7d",
-          "30d"
-        ]
-      },
-      "timezone": "",
-      "title": "RHTAP SLOs",
-      "uid": "rhtap-slos",
-      "version": 2,
-      "weekStart": ""
+      "title": "SLO Template",
+      "type": "row"
     }
+  ],
+  "refresh": "1h",
+  "schemaVersion": 37,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "current": {
+          "selected": true,
+          "text": "rhtap-observatorium-production",
+          "value": "rhtap-observatorium-production"
+        },
+        "hide": 0,
+        "includeAll": false,
+        "multi": false,
+        "name": "datasource",
+        "options": [],
+        "query": "prometheus",
+        "queryValue": "",
+        "refresh": 1,
+        "regex": "/^rhtap-/",
+        "skipUrlSync": false,
+        "type": "datasource"
+      }
+    ]
+  },
+  "time": {
+    "from": "now-1h",
+    "to": "now"
+  },
```

Comment:
Please don't modify the overall dashboard settings. We should use the default 28d SLO measurement window.

Reply:
Undid modifications in overall dashboard settings